docs/content/preview/drivers-orms/go/_index.md

```diff
@@ -41,16 +44,4 @@ Visual Studio Code for developing your Go project.
 
 Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Create a free cluster](../../yugabyte-cloud/cloud-basics/create-clusters-free/).
 
-Alternatively, set up a standalone YugabyteDB cluster by following the steps in [Install YugabyteDB](/preview/quick-start/install/macos). 
-
-## Usage examples
```

Comment:
Looks like we are removing the `Usage examples` section from all the drivers landing page. Wondering if it's getting removed because it is redundant?

Reply:
Examples moved to table at top